<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings</class>
 <widget class="QDialog" name="settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Установка стиля</string>
  </property>
  <widget class="QPushButton" name="pushButton_brush_style">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>изменить стиль</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_pen_style">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>изменить стиль</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_pen">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Обводка:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_pen_color">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>изменить цвет</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_pen_width">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>150</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>изменить толщину</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_brush">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Заливка:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_brush_color">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>изменить цвет</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>220</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Принять</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
